modules/ROOT/pages/single-sign-on-with-cas.adoc

```diff
@@ -208,8 +208,8 @@ The default `AuthenticationFilter` that manages CAS authentication applies only
 
 REST API are not part of them, but if an http session already exists thanks to cookies, REST API can be used.
```

Comment:
an http session already exists =>an https session already exists
 are we sure its not https?

Reply:
Um, I would let @abirembaut answer that since I'm not very good with https.

Reply:
No, the technical name is HTTP session even if we are in HTTPS. It would be good to put it in uppercase though

Reply:
Done